pkg/controllermanager/controller/managedseedset/replica_test.go

```diff
@@ -373,11 +367,7 @@ var _ = Describe("Replica", func() {
 									Object: &gardenletv1alpha1.GardenletConfiguration{
 										SeedConfig: &gardenletv1alpha1.SeedConfig{
 											SeedTemplate: gardencorev1beta1.SeedTemplate{
-												Spec: gardencorev1beta1.SeedSpec{
-													DNS: gardencorev1beta1.SeedDNS{
-														IngressDomain: pointer.String("ingress." + replicaName + ".example.com"),
```

Comment:
Why was this expectation not replaced by a proper `.spec.ingress.domain` expectation? Earlier, this test expected a calculated `IngressDomain` field which does not happen anymore.

Reply:
Nice catch, adapted.